src/policy/fees.cpp

```diff
@@ -432,7 +432,7 @@ void TxConfirmStats::Read(CAutoFile& filein, int nFileVersion, size_t numBuckets
     maxPeriods = confAvg.size();
     maxConfirms = scale * maxPeriods;
 
-    if (maxConfirms <= 0 || maxConfirms > 6 * 24 * 7) { // one week
+    if (maxConfirms > 6 * 24 * 7) { // one week
```

Comment:
`maxConfirms == 0` test is missed even for type `size_t`.
Agree with @gmaxwell.